packages/fxa-content-server/app/tests/spec/views/connect_another_device.js

```diff
@@ -301,27 +305,6 @@ describe('views/connect_another_device', () => {
         });
       });
 
-      it('shows FxDesktop text, pairing CTA', () => {
```

Comment:
This test was testing a set of conditions that shouldn't happen (and we don't support anymore).  `canSignIn` is checking for webchannel support and signed in state.  If the user is using a supported version of Firefox (anything remotely recent), webchannel is available, so we cannot have a state where the browser is desktop Firefox, the user is not signed in, _and_ cannot sign in.